src/main/java/tds/exam/results/messaging/ExamCompletedMessageListener.java

```diff
@@ -70,7 +71,11 @@ public void handleMessage(final String examId) {
     private void processMessage(final String examId) {
         // Once this ERT message has been received, the exam status is "submitted"
         final UUID id = UUID.fromString(examId);
-        examResultsService.findAndSendExamResults(id);
-        examService.updateStatus(id, ExamStatusCode.STATUS_SUBMITTED);
+        TDSReport report = examResultsService.findAndSendExamResults(id);
+
+        //Expired Exams can have TRT's created.  However, those statuses do not get updated.
+        if(!report.getOpportunity().getStatus().equals(ExamStatusCode.STATUS_EXPIRED)) {
+            examService.updateStatus(id, ExamStatusCode.STATUS_SUBMITTED);
+        }
```

Comment:
Long term it may be worth reconsidering this design a bit as the status logic is spread across this and exam service.  It may be worth considering adding a new endpoint in exam service that contains the logic of what to do with an exam after the ERT has submitted it.

Something like a `/exams/{id}/reported` that would indicate that the exam had been reported for scoring.  Similar to the webhook TIS has in its system that calls back to ERT.